### ISSUE TITLE
Bytes32.wrap(Bytes value, int offset) doesn't honor offset for specific Bytes param

### ISSUE THREAD

**Issue by Nashatyrev** [apache/incubator-tuweni#168]
Issue in the implementation of the `Bytes32.wrap(Bytes value, int offset)` method:
```java
    Bytes bytes = Bytes.fromHexString(
        "0x00112233445566778899aabbccddeeff00112233445566778899aabbccddeeff00112233445566778899aabbccddeeff");
    MutableBytes mutableBytes = MutableBytes.create(48);
    bytes.copyTo(mutableBytes);
    System.out.println(Bytes32.wrap(mutableBytes, 1));
```
Expected output: 
```
0x112233445566778899aabbccddeeff00
```
Actual output: 
```
0x00112233445566778899aabbccddeeff
```

**Comment by atoulme** [comment]
Which version are you using?
I have reproduced the issue. I recommend you use this notation as workaround:
`mutableBytes.slice(1, 32);`

**Comment by Nashatyrev** [comment]
We are using `1.0.0` at the moment. 
Yep, thanks for the workaround, I believe this is exactly what I've used

**Comment by Nashatyrev** [comment]
I believe this was fixed with #179
